### DISCUSSION TITLE
Can I add new files to the directory while Fscrawler is still indexing the previous ones?

### DISCUSSION THREAD

**Discussion by ScottCov** [dadoonet/fscrawler#1810] [General]
Hi still a little unclear....I am using the docker container....is it ok add a new batch of files to my directory while Fscrawler is still indexing older ones so long as I touch the new files?

**Comment by dadoonet** [comment]
Yes you can.
Worse case is that they won't be picked up within the current run but they should for the next run.

Unless there's a bug.
